package-lock.json

```diff
@@ -0,0 +1,12739 @@
+{
```

Comment:
If the repository already has a yarn.lock file, you should keep using yarn instead of npm. Avoid having yarn.lock and package-lock.json in the same repository at the same time.